src/erlang_ls_server.erl

```diff
@@ -21,12 +21,16 @@
         , handle_cast/2
         ]).
 
-%% Notifications API
+%% API
 -export([ process_requests/2
         , set_connection/2
         , send_notification/3
         ]).
 
+%% Testing
```

Comment:
I added this function to simplify the setup we had in the proper tests. Just resetting the internal state makes everything much much simpler.